lib/index.js

```diff
@@ -141,6 +149,13 @@ module.exports.createClient = function({
         .then(_ => {
           return client.api;
         });
+    },
+
+    async cleanup() {
+      await client.queue.waitForCompletion();
+      client.queue.empty();
+      client.queue.reset();
```

Comment:
Do we need to reset the queue?

Reply:
Added it for good measure. `queue.reset()` is getting called from terminate inside testsuite, I want to do something similar.. probably I should add a timeout for the `waitFroCompletion` method, and also call `removeAllListeners`.

Reply:
done